Description: Fix build issues on Sparc
 This patch fixes build issues on Sparc by adding the necessary
 assembly routines for VT.
 .
 It can be removed when 1.3.4 is released.
Forwarded: https://svn.open-mpi.org/trac/ompi/ticket/1973
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=519725
Author: Manuel Prinz <manuel@debian.org>
Last-Update: 2009-10-13

--- a/ompi/contrib/vt/vt/vtlib/vt_pform_linux.c
+++ b/ompi/contrib/vt/vt/vtlib/vt_pform_linux.c
@@ -172,6 +172,8 @@
     clock_value = __getReg(_IA64_REG_AR_ITC);
 # elif defined(__alpha__)
     asm volatile ("rpcc %0" : "=r" (clock_value));
+# elif defined(__sparc__)
+    asm ("rd %%tick, %0" : "=r" (clock_value));
 # else
     /* ... TSC */
     {
